srcpkgs/gimp/patches/gtk-workaround.patch

```diff
@@ -0,0 +1,40 @@
+--- a/app/widgets/gimpdeviceinfo-coords.c
++++ b/app/widgets/gimpdeviceinfo-coords.c
```

Comment:
where is this patch from? Could you add a source to the header?

Reply:
The link is in the description of the commit:

https://github.com/void-linux/void-packages/pull/47014/commits/b4ae183b7111fa153640d0eb9c117963ec51440b

Reply:
You should leave the patch header in. (E.g. the commit hash, the commit name, the commit date, the commit author, the commit description, etc)

Reply:
> The link is in the description of the commit:
> 
> [b4ae183](https://github.com/void-linux/void-packages/commit/b4ae183b7111fa153640d0eb9c117963ec51440b)

Please avoid truncate the header text of patch as it shows the referance to source location.

Reply:
I removed the "info part" for 2 reasons:

1. I wan't sure if the building process wouldn't fail with it in it.
2. I modeled the patch after the patches that were already present and they didn't have such a long header.

I fixed it now. Hope it worked alright.